### PR TITLE
Added Medication Schema + Migration

### PR DESCRIPTION
**Pull request by kpirabaharan**
- Postgresql
- GORM (ORM)
- Gin Framework

### CODE REVIEW

**Review by Devanand-Sharma — APPROVED** [review]

File: cmd/server/main.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"github.com/Devanand-Sharma/medication-dispenser-backend/pkg/handlers"
```
[the remark this answers is not in the document]
Comment (kpirabaharan) [reply cmd/server/main.go:4]: Yes

File: internal/handlers/medication.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/Devanand-Sharma/medication-dispenser-backend/pkg/database"
+	"github.com/Devanand-Sharma/medication-dispenser-backend/pkg/models"
```
[the remark this answers is not in the document]
Comment (kpirabaharan) [reply internal/handlers/medication.go:8]: Yes

File: internal/database/database.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	"gorm.io/gorm"
+)
+
+// Connect to the database and return instance
+func Connect() *gorm.DB {
+	// Load Config
+	cfg := database.LoadConfig()
+
+	// Print Config
+	// fmt.Printf("DBUser: %v, DBPassword: %v, DBName: %v, DBHost: %v, DBPort: %v\n", cfg.DBUser, cfg.DBPassword, cfg.DBName, cfg.DBHost, cfg.DBPort)
```
[the remark this answers is not in the document]
Comment (kpirabaharan) [reply internal/database/database.go:19]: Yes

File: internal/models/medication.go [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+type Medication struct {
+	gorm.Model
+	Name         string          `gorm:"not null"`
+	Condition    string          `gorm:"not null"`
+	Route        MedicationRoute `gorm:"not null"`
+	Dose         int             `gorm:"not null"`
+	Prescription Prescription
+	Dosage       Dosage
+	Instructions string
+	Image        string
```
[the remark this answers is not in the document]
Comment (kpirabaharan) [reply internal/models/medication.go:93]: Ok

File: internal/handlers/medication.go [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+func ListMedications(c *gin.Context) {
+	db := database.Connect()
+
+	var medications []models.Medication
+	db.Find(&medications)
+
+	// Print the medications in console
+	for _, medication := range medications {
+		fmt.Println(medication)
```
[the remark this answers is not in the document]
Comment (kpirabaharan) [reply internal/handlers/medication.go:20]: This is to show medications works for now. We will get medications by user and will look different. We can update it later.

**Review by PranayjitSingh — APPROVED** [review]

**Pull request merged**